docs/client-side/yeoman.rst

```diff
@@ -0,0 +1,188 @@
+
+The Yeoman command line tool **yo** works alongside a Yeoman generator. Generators define the technologies that go into a project. Here are a few sample `generators <http://yeoman.io/generators/>`_:
+
+- `AngularJS Generator <https://github.com/yeoman/generator-angular>`_ creates a starting point for a new single-page Angular application. 
+- `JQuery Generator <https://github.com/yeoman/generator-jquery>`_ creates the shell code for a jQuery plug-in.
+- `Chrome App Generator <https://github.com/yeoman/generator-chromeapp>`_ generates everything you need to create an extension for the Chrome browser.
+- The `Flux-React generator <https://github.com/banderson/generator-flux-react>`_ creates an application based on Facebook's Flux/React architecture.
+- `ASP.NET generator <https://www.npmjs.com/package/generator-aspnet>`_ creates ASP.NET 5, DNX projects.
+- You can even `generate your own Yeoman generator <https://github.com/yeoman/generator-generator>`_.
+
+In this article:
+	- `Getting Started`_
+	- `Building and Running from Visual Studio`_
+	- `Client-Side Build Support`_
+	- `Restoring, Building and Hosting from the Command Line`_
+	- `Adding to Your Project with Sub Generators`_
+	
+Getting Started
+---------------
+The `ASPNET generator <https://www.npmjs.com/package/generator-aspnet>`_ creates ASP.NET 5, DNX projects that can 
+be loaded into Visual Studio 2015 or run from the command line. The generator creates the following project types:
+  
+- Empty Application: An application with minimal dependencies that outputs "hello world" to the browser.     
+- Console Application: Another minimal application that outputs "hello world" to the console.
+- Web Application: A complete MVC web application with a simple home page and examples for managing accounts and login.  
+- Web API Application: An ASP.NET 5 web service with one controller.   
+- Nancy ASP.NET Application: A light-weight HTTP service with one module.
+- Class Library: Creates a DLL that can be referenced by other ASP.NET projects.
+
+This walk-through demonstrates how to use Yeoman to generate an ASP.NET 5 web application. 
+ 
+1.  Follow the instructions on http://yeoman.io/learning/ to install **yo** and other required tools. 
+
+2.	On the command line, install the ASP.NET generator:  
+
+.. code-block:: console
+
+ npm install -g generator-aspnet
+
+.. note:: The **–g** flag installs the generator globally so that you can use it from any path on your system.
+ 
+3.	Make a new directory where your project will be generated:
+
+.. code-block:: console
+
+ mkdir c:\MyYo
+
+4.	On the command line, make the new directory the current directory.
+
+.. code-block:: console
+ 
+ cd c:\MyYo
+
+5.	Run the ``yo`` command and pass the name of the generator.
+
+.. code-block:: console 
+
+ yo aspnet
+
+6.	The aspnet generator displays a menu. Select the **Web Application** and press ``Enter``.
+
+.. image:: yeoman/_static/yeoman-yo-aspnet.png
```

Comment:
Looks like the background coloring in PowerShell doesn't play nice with Yeoman. Can we get a new image from a simple cmd prompt?